### DISCUSSION TITLE
var dumper error:

### DISCUSSION THREAD

**Discussion by remco-pc** [symfony/symfony#42396] [Q&A]
<br />
<b>Fatal error</b>:  Class symfony\component\vardumper\caster\conststub wasn't preloaded in <b>/mnt/c/Repository/.../vendor/symfony/var-dumper/Caster/ConstStub.php</b> on line <b>21</b><br />

**Comment by derrabus** [comment]
Hello and thank you for reaching out.

This tracker is meant for tracking bugs and feature requests. If you are looking for support, you'll get faster responses by using one of our official support channels:

* GitHub Discussions: https://github.com/symfony/symfony/discussions
* StackOverflow: https://stackoverflow.com/questions/tagged/symfony
* Slack: https://symfony.com/slack

See https://symfony.com/support for more support options.

**Reply by xabbuh** [reply]
@derrabus we can transform an issue into a discussion :)

**Comment by lyrixx** [comment]
Can you give us more information ?
A trace, a reproducer?

**Comment by javiereguiluz** [comment]
@remco-pc please, provide any additional information so we can reproduce this error. Otherwise we'll need to close the discussion. Thanks!
